internal/validations/observability/outputs/validate_url_to_output_tls.go

```diff
@@ -19,13 +19,13 @@ func validateURLAccordingToTLS(output obs.OutputSpec) (results []string) {
 	case obs.OutputTypeHTTP:
```

Comment:
Can we use URL for other output types too?

Reply:
reverting URL and adding marker to all URLs